packages/headless/src/composables/tooltip/createTooltip.ts

```diff
@@ -106,6 +107,9 @@ export const createTooltip = createBuilder((options: CreateTooltipOptions) => {
 
   return {
     elements: {
+      root: {
```

Comment:
Technically this is a breaking change right? Because if you used this composable, you need to change your code to bind the root element which was not needed before

Reply:
Updated the Changeset